src/components/layout/header/LocaleMenu.tsx

```diff
@@ -20,9 +19,8 @@ function LocaleMenu({ t }: { t: { changeLocale: string } }) {
   return (
     <DropdownMenu>
       <DropdownMenuTrigger asChild>
-        <Button variant='ghost' size='icon'>
-          <Globe2Icon className='h-[1.2rem] w-[1.2rem]' />
-          <span className='sr-only'>{t.changeLocale}</span>
+        <Button variant='ghost' size='icon' title={t.changeLocale}>
+          <Globe2Icon className='h-[1.2rem] w-[1.2rem]' aria-hidden />
```

Comment:
same as other places

Reply:
Also when using the title attribute, you have to also use aria label. They have to be set to the same. This is because title is not used as aria label as expected. Good that you removed sr-only span since aria-label is preferred